apps/exports/models.py

```diff
@@ -132,12 +138,14 @@ def mark_skipped(self):
         self.completed_at = timezone.now()
         self.save()
 
-class ExportRun(ExportRunBase):
-    export_config = models.ForeignKey(ExportConfig, on_delete=models.CASCADE, related_name='runs')
 
+class ExportRun(ExportRunBase):
+    base_export_config = models.ForeignKey(ExportConfig, on_delete=models.CASCADE, related_name='runs', null=True)
+    export_config_version = models.ForeignKey(Version, on_delete=models.CASCADE, null=True)
```

Comment:
I added this additional field instead of replacing the existing one and kept both in order to keep access between Configs and Runs easier.